torchvision/models/vision_transformer.py

```diff
@@ -25,6 +26,16 @@
 }
 
 
+class ConvStemConfig(NamedTuple):
+    out_channels: int
+    kernel_size: int
+    stride: int
+    padding: int
+    norm_layer: nn.Module = nn.BatchNorm2d
```

Comment:
When you get issues with mypy, it's useful for look for similar patterns in the codebase. This type is wrong here, it's a callable that will return a Module. Aka:
```suggestion
    norm_layer: Callable[..., nn.Module] = nn.BatchNorm2d
```